### PR TITLE
lsof: Fix build for Linux

### PR DESCRIPTION
**Pull request by iMichka**
and capitalize manual entry as the file has a capital L

- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Review by issyl0 — DISMISSED** [review]

File: Formula/lsof.rb [hunk 1]
```diff
@@ -16,23 +16,29 @@ class Lsof < Formula
...
-
     mv "00README", "README"
-    system "./Configure", "-n", "darwin"
+    system "./Configure", "-n", os
 
     system "make"
     bin.install "lsof"
-    man8.install "lsof.8"
+    man8.install "Lsof.8"
     prefix.install_metafiles
```
Comment (carlocab) [inline Formula/lsof.rb:42]:
```suggestion
```
Since we're here already let's remove this now as it's no longer needed.

Comment (iMichka) [reply Formula/lsof.rb]: If you say so :) I have no clue what this does/did.

Comment (carlocab) [reply Formula/lsof.rb]: Installs the metafiles, obviously 😄

Comment (iMichka) [reply Formula/lsof.rb]: I mean, it installed metafiles, and now the metafiles are not needed. That doest not tell me why this needs a change 😄

Comment (carlocab) [reply Formula/lsof.rb]: Oh it's no longer needed because `brew` now installs the metafiles without being told.

**Review by carlocab — APPROVED** [review]

**Pull request merged**